doc/weakly_supervised.rst

```diff
@@ -528,23 +602,185 @@ points, while constrains the sum of distances between dissimilar points:
   .. [2] Adapted from Matlab code `here <http://www.cs.cmu
      .edu/%7Eepxing/papers/Old_papers/code_Metric_online.tar.gz>`_.
 
+
+.. _learning_on_quadruplets:
+
 Learning on quadruplets
 =======================
 
-A type of information even weaker than pairs is information about relative
-comparisons between pairs. The user should provide the algorithm with a
-quadruplet of points, where the two first points are closer than the two
-last points. No target vector (``y``) is needed, since the supervision is
-already in the order that points are given in the quadruplet.
+
+
+The goal of weakly-supervised metric-learning algorithms is to transform
```

Comment:
this is irrelevant here
should be replaced by a quick paragraph in the same spirit as for the section on pairs